e2e/cypress/integration/enterprise/guest_accounts/guest_removal_ui_spec.js

```diff
@@ -32,7 +32,7 @@ function removeUserFromAllChannels(verifyAlert) {
         // * Verify if guest user gets a message when the channel is removed. Does not appears when removed from last channel of the last team
         if (channel === 'Town Square' || verifyAlert) {
             cy.get('#removeFromChannelModalLabel').should('be.visible').and('have.text', `Removed from ${channel}`);
-            cy.get('.modal-body').should('be.visible').and('have.text', `sysadmin removed you from ${channel}`);
+            cy.get('.modal-body').should('be.visible').contains(`removed you from ${channel}`);
```

Comment:
The remover is sometimes `sysadmin` or `Someone` depending whether the remover is initially loaded or not.